doc/source/configuring.rst

```diff
@@ -75,6 +75,23 @@ The rules are organized by mailbox, and then listed in order. The
 first rule that matches a message triggers the associated action, and
 then processing for that message stops.
 
+Expiration Rules
+----------------
+
+An Expiration ``expired`` rule is initialized by specifying the 'age', 
+number of days for the email to "live" in the specified mailbox. 
+If the message(s) has/have been in the mailbox for more than the 
+specified number of days without being moved to a destination area, 
+it will be moved to the "Trash" directory. If age = 0, the rule is 
```

Comment:
The rule itself isn't tied to the trash action. That's up to the user to configure. So it's more accurate to say something like "if the message date is older than the specified age, the rule will match the message" and leave the details about what happens next out of it.